clients/python-wrapper/lakefs/branch.py

```diff
@@ -130,45 +131,51 @@ def object(self, path: str) -> WriteableObject:
 
         return WriteableObject(self.repo_id, self._id, path, client=self._client)
 
-    def uncommitted(self, max_amount: Optional[int], after: str = '', prefix: str = '') -> Generator[Change]:
+    def uncommitted(self, max_amount: Optional[int], after: Optional[str] = None, prefix: Optional[str] = None,
+                    **kwargs) -> \
+            Generator[Change]:
         """
-        List uncommitted changes
+        Returns a diff generator of uncommitted changes on this branch
 
-        :param max_amount: maximum amount of changes to return
-        :param after: return changes after this path
-        :param prefix: return changes with this prefix
-        :return: a generator of Change objects
+        :param max_amount: Stop showing changes after this amount
+        :param after: Return items after this value
+        :param prefix: Return items prefixed with this value
         :raises:
             NotFoundException if branch or repository do not exist
             NotAuthorizedException if user is not authorized to perform this operation
             ServerException for any other errors
         """
+
         for diff in self._get_generator(self._client.sdk_client.branches_api.diff_branch,
-                                        self._repo_id, self._id, max_amount=max_amount, after=after, prefix=prefix):
+                                        self._repo_id, self._id, max_amount=max_amount, after=after, prefix=prefix,
+                                        **kwargs):
             yield Change(**diff.dict())
 
-    def import_data(self, commit_message: str) -> ImportManager:
+    def import_data(self, commit_message: str, metadata: dict = None) -> ImportManager:
         """
         Import data to lakeFS
 
         :param commit_message: once the data is imported, a commit is created with this message
         :return: an ImportManager object
         """
-        return ImportManager(self._repo_id, self._id, commit_message, self._client)
+        return ImportManager(self._repo_id, self._id, commit_message, metadata, self._client)
 
-    def delete_objects(self, object_paths: str | Iterable[str]) -> None:
+    def delete_objects(self, object_paths: str | Iterable[str | StoredObject]) -> None:
```

Comment:
If you already allowed it:
```suggestion
    def delete_objects(self, object_paths: str | StoredObject | Iterable[str | StoredObject]) -> None:
```